api-outbound.yaml

```diff
@@ -288,14 +291,17 @@ components:
     AssetEventWithHostname:
       type: object
       required:
```

Comment:
Task:  Should "scanTime" be part of the required outbound fields here?  I think it should...